book.rb

```diff
@@ -0,0 +1,8 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+  end
+end
```

Comment:
- Kindly, students created the `many-to-many` (also `has-many-through`) relationship between `Person` and `Book` using the intermediate class `Rental` :+1: 
```ruby
class Book
  attr_accessor :title, :author, :rentals
  .
  .

  def initialize(title, author)
    .
    .
    @rentals = []
  end

  def add_rental(person, date)
    Rental.new(date, self, person)
  end
end
```